generators/client/templates/angularjs/src/main/webapp/app/components/language/_language.filter.js

```diff
@@ -36,6 +36,7 @@
                 'en': 'English',
                 'es': 'Español',
                 'et': 'Eesti',
+                'fa': 'فارسی',
```

Comment:
It would be better to change the pipe to below and then use it in the above service. 
```
(function() {
     'use strict';
 
     angular
         .module('<%=angularAppName%>')
         .filter('findLanguageFromKey', findLanguageFromKey);
         .filter('findLanguageRtlFromKey', findLanguageRtlFromKey);
     var languages = {
        'ca': { name: 'Català' },
        ....
        'fa': { name: 'فارسی', rtl: true },
        ...
        'zh-tw': { name: '繁體中文' }
     }
     function findLanguageFromKey() {
         return findLanguageFromKeyFilter;
 
         function findLanguageFromKeyFilter(lang) {
             return languages[lang].name;
         }
     }
     function findLanguageRtlFromKey() {
         return findLanguageRtlFromKeyFilter;
 
         function findLanguageRtlFromKeyFilter(lang) {
             return languages[lang].rtl;
         }
     }
 })();
```